index.js

```diff
@@ -0,0 +1,84 @@
+console.log("hello, vanilla.");
+var howmany = document.getElementById('try');
+var flag = 0;
+var startbtn = document.getElementById("startbtn");
+
+var count=0;
+var strike=0;
+var ball=0;
+var input=0;
+var opportunity=10;
+
+var answer; 
+
+function setanswer(){
```

Comment:
```suggestion
function setAnswer(){
```

Reply:
```suggestion
function setAnswer(){
```
함수 네이밍에는 carmelCase를 사용해주세요.
* 참고 - [naming--camelCase](https://github.com/airbnb/javascript#naming--camelCase)